index.js

```diff
@@ -102,7 +102,12 @@ export default class Carousel extends Component {
         /**
          * Fired when snapping to an item
          */
-        onSnapToItem: PropTypes.func
+        onSnapToItem: PropTypes.func,
+        /**
+         * If you want to set the side margin of the scroller
+         * This is not optimal, I may or may not work in your case
+         */
+        carouselSideMargin: PropTypes.number,
```

Comment:
@skeie My opinion is that this prop should be renamed `carouselHorizontalPadding`, as it better describes its effect. What do you think?